.eslintignore

```diff
@@ -1,3 +1,4 @@
 **/node_modules/*
 **/dist/*
 .github/actions/**/index.js"
+/scripts/**
```

Comment:
I think because these are node scripts and have different requirements/environment so existing ESLint rules might not make sense so I decided to exclude them from linting for now

Reply:
I don't think we should ignore them. Node scripts are just like any other JS script and the same formatting and styles should apply to them. We've seen this in the GitHub actions as well. It makes sense that JS code everywhere is consistent, regardless of the platform it's running on.